tests/console/pulpito.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run pulpito the teuthology result webpage and tear down openstack when test passed
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base 'x11test';
+use strict;
+use testapi;
+
+sub run {
+    select_console 'x11';
+    x11_start_program('firefox http://pulpito.suse.de:8081', valid => 0);
+    assert_screen 'pulpito-dashboard';
+    # tab over web page to get to pulpito-test-list
+    for (1 .. 22) { send_key 'tab' }
```

Comment:
Does this work stably? I would recommend to use send_key_until_needle_match here. Or as it's x11 assert_and_click.

Reply:
since when is assert_and_click stable ? I want to avoid any additional needle, when this doesn;t work well I will change it

Reply:
If you will run it only on qemu on x86 that's fine (I guess), but we had a lot of issues when it will break because single key was missed. I have not seen any issues with assert_and_click, so I would prefer it to magic number of key presses as we haven't moved further to use selenium for web UIs.

Reply:
I have seen and experienced assert_and_click issues, I was surprised that this tab worked so well for me, maybe in osd it will not, let's see. For selenium I will have to use some custom installed chrome